guides/source/api_app.md

```diff
@@ -151,6 +151,113 @@ This will do three main things for you:
 - Configure the generators to skip generating views, helpers, and assets when
   you generate a new resource.
 
+### Generating a New Resource
+
+To see how our newly created API handles generating a new resource, let's create
+new Group and Member resources. Each group will have a name, and each member will
+have an associated group_id and name column.
+
+```bash
+$ bin/rails g scaffold Group name:string
+$ bin/rails g model Member name:string group:references
+```
+
+Before we can use our scaffolded code, we need to update our database scheme.
+
+```bash
+$ bin/rails db:migrate
+```
+Our models should be set up as:
+```ruby
+# app/models/group.rb
+class Group < ApplicationRecord
+  has_many :members
+end
+```
+```ruby
+# app/models/member.rb
+class Member < ApplicationRecord
+  belongs_to :group
+end
+```
+
+Now if we open our Groups Controller, we should notice that with an API rails
+app we are rendering JSON data only. On the index action we query for Group.all
+and assign it an instance variable called `@groups`. Passing it to `render` with the
+`:json` option will automatically render the groups as JSON.
+
+```ruby
+# app/controllers/groups_controller.rb
+class GroupsController < ApplicationController
+  before_action :set_group, only: %i[ show update destroy ]
+
+  # GET /groups
+  def index
+    @groups = Group.all
+
+    render json: @groups
+  end
+
+  # GET /groups/1
+  def show
+    render json: @group
+  end
+
+  # POST /groups
+  def create
+    @group = Group.new(group_params)
+
+    if @group.save
+      render json: @group, status: :created, location: @group
+    else
+      render json: @group.errors, status: :unprocessable_entity
+    end
+  end
+
+  # PATCH/PUT /groups/1
+  def update
+    if @group.update(group_params)
+      render json: @group
+    else
+      render json: @group.errors, status: :unprocessable_entity
+    end
+  end
+
+  # DELETE /groups/1
+  def destroy
+    @group.destroy
+  end
+
+  private
+    # Use callbacks to share common setup or constraints between actions.
+    def set_group
+      @group = Group.find(params[:id])
+    end
+
+    # Only allow a list of trusted parameters through.
+    def group_params
+      params.require(:group).permit(:name)
+    end
+end
+```
+
+Finally we can add some groups to our database.
+
+```bash
+bin/rails console
+
+Group.create(name: "Rails Founders")
+Group.create(name: "Rails Contributors")
+```
+
+With some data in the app, we can boot up the server and see our JSON data.
+```bash
+bin/rails server
+```
+```json
+[{"id":1,"name":"Rails Founders","created_at":"yyyy-MM-ddTHH:mm:ss.SSZ","updated_at":"yyyy-MM-ddTHH:mm:ss.SSZ"},{"id":2,"name":"Rails Contributors","created_at":"yyyy-MM-ddTHH:mm:ss.SSZ","updated_at":"yyyy-MM-ddTHH:mm:ss.SSZ"}]
```

Comment:
I think it's more readable if we add some linkbreaks (even if those aren't present in the actual JSON).
```suggestion
[
{"id":1, "name":"Rails Founders", "created_at":" ...},
{"id":2, "name":"Rails Contributors", "created_at": .... }
]
```